pkg/cmd/openfaas_ingress_app_test.go

```diff
@@ -0,0 +1,68 @@
+package cmd
+
+import (
+	"io/ioutil"
+	"testing"
+)
+
+func Test_build_yaml_returns_correct_substitutions(t *testing.T) {
+	got := string(buildYaml("openfaas.subdomain.example.com", "openfaas@subdomain.example.com"))
+	if want != got {
+		t.Errorf("suffix, want:%s, got:%s", want, got)
+	}
+}
+
+
+var want = `
+apiVersion: extensions/v1beta1 
+kind: Ingress
+metadata:
+  name: openfaas-gateway
+  namespace: openfaas
+  annotations:
+    certmanager.k8s.io/acme-challenge-type: http01
```

Comment:
Did this annotation also move to the `cert-manager.io` namespace? (I'm not sure.